src/libraries/intellisense.targets

```diff
@@ -17,7 +17,11 @@
     <TargetFramework>$(TargetFramework.SubString(0, $(TargetFramework.IndexOf('-'))))</TargetFramework>
   </PropertyGroup>
 
-    <!-- Split $(TargetFramework) (e.g. net45) into short identifier and short version (e.g. 'net' and '45'). -->
+  <PropertyGroup>
+    <RefPath>$([MSBuild]::NormalizeDirectory('$(RefRootPath)', '$(TargetFramework)'))</RefPath>  
```

Comment:
Why not instead set `BuildTargetFrameworkRefPath`? It seems like `RefPath` takes its value:
https://github.com/dotnet/runtime/blob/952eecabbdd57540f4fe72c490c794e79c3d0cf3/src/libraries/Directory.Build.props#L257

Reply:
When we are in a VS, and we want to run tests or build stuff for net472 or some other framework than net50, there is no way to override the default value of BuildTargetFramework.
Refpaths for all the designTimeBuilds will also have the netcoreapp50 refpath only.

Reply:
I mean setting `BuildTargetFrameworkRefPath` here instead of `RefPath`. Does Directory.Build.targets come into play in DesignTime build?

Reply:
@safern BuildTargetFrameworkRefPath is the value for the vertical, not the current configuration.  BuildTargetFrameworkRefPath doesn’t need to be reset: it’s value remains the same and is only set by root level build invocation. RefPath on the other hand is dependent on project TargetFramework and needs to be reset here since.

Reply:
I see, I missed the condition 😄